### ISSUE TITLE
Conversational card does not exist anymore

### ISSUE THREAD

**Issue by mpalmier74** [actions-on-google/codelabs-nodejs#77]
At link: https://codelabs.developers.google.com/codelabs/actions-1#2
guide says:

> 4 Click Create project.
> 5 Rather than pick a category, scroll down to the More options section and click on the Conversational card.

but "Conversational" card does not exist anymore. Guide may need update

![image](https://user-images.githubusercontent.com/13572439/121659208-5e449e00-caa2-11eb-8c1f-b588bc3b5206.png)

**Comment by Fleker** [comment]
Yeah that may need an update. "Custom" is the right choice for you.
